plugin/editor/blocks/text-block/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/elements';
+import { registerBlock, Editable } from '@wordpress/blocks';
+
+registerBlock( 'wp', 'Text', {
+	edit( state, onChange ) {
+		return createElement( Editable, {
+			value: state.value,
+			onChange: ( value ) => onChange( { value } )
+		} );
+	},
+	save( state ) {
+		return createElement( 'p', null, state.value );
```

Comment:
>do you think it's worth it to use an element here instead of just a string?

Depends whether we want to support multiple return types or not. In this case we assume the `save` function must return of type "Element". String is perfectly suitable, but we'd need to build support for it and then question whether overloading the method return type is really benefiting us more than the cost of confusion around supporting multiple types.